src/main/kotlin/fi/metatavu/timebank/api/controllers/SynchronizeController.kt

```diff
@@ -73,6 +103,33 @@ class SynchronizeController {
         }
     }
 
+    /**
+     * Synchronizes deleted time entries from Forecast to Time-bank
+     *
+     * @param after YYYY-MM-DD LocalDate
+     * @param forecastPersons list of ForecastPersons
+     * @param forecastTimeEntries list of ForecastTimeEntries
+     */
+    suspend fun synchronizeDeletions(after: LocalDate? = oldestSyncDate, forecastPersons: List<ForecastPerson>, forecastTimeEntries: List<ForecastTimeEntry>) {
```

Comment:
This methods after parameter doesn't need the default value due to the new `startSynchronization` method already defaulting it to `oldestSyncDate`